src/quo2/components/inputs/locked_input/view.cljs

```diff
@@ -0,0 +1,30 @@
+(ns quo2.components.inputs.locked-input.view
+  (:require [react-native.core :as rn]
+            [status-im2.contexts.quo-preview.tabs.segmented-tab]
+            [quo2.foundations.colors :as colors]
+            [status-im.ui.components.icons.icons :as icons]
+            [quo2.components.inputs.locked-input.style :as style]
+            [quo2.theme :as theme]
+            [quo2.components.markdown.text :as text]))
+
+(defn- info-box
+  [theme icon label-text]
+  [rn/view
+   {:style (style/info-box-container theme)}
+   [rn/view {:margin-right 1}
```

Comment:
Is there any reason for adding `margin-right 1` here and using `margin-left 7` in the text?